dpnp/backend/tests/test_main.cpp

```diff
@@ -57,6 +57,8 @@ int main(int argc, char** argv)
 
     // currently using global queue
 
+    dpnp_queue_initialize_c(QueueOptions::GPU_SELECTOR);
```

Comment:
this is not correct usage.

Reply:
It's just needed to reproduce the issue.

Reply:
I'm not going to merge the PR.

Reply:
see https://github.com/IntelPython/dpnp/pull/661#discussion_r601578733